web/tests/pm_list_data.test.js

```diff
@@ -211,9 +217,23 @@ test("get_list_info_unread_messages", ({override}) => {
     pm_conversations.recent.insert([zoe.user_id, bob.user_id, alice.user_id], 8);
     pm_conversations.recent.insert([cardelio.user_id, zoe.user_id], 9);
     pm_conversations.recent.insert([cardelio.user_id, bob.user_id], 10);
+    pm_conversations.recent.insert([cardelio.user_id, alice.user_id], 11);
+    pm_conversations.recent.insert([cardelio.user_id, zoe.user_id, bob.user_id], 12);
+    pm_conversations.recent.insert([cardelio.user_id, zoe.user_id, alice.user_id], 13);
+    pm_conversations.recent.insert([cardelio.user_id, bob.user_id, alice.user_id], 14);
+    pm_conversations.recent.insert([cardelio.user_id, bob.user_id, alice.user_id, zoe.user_id], 15);
+    pm_conversations.recent.insert([cardelio.user_id], 16);
+    pm_conversations.recent.insert([iago.user_id], 17);
```

Comment:
In retrospect, it might have been more clever to make these tests be able to change the 8/15 constants for testing of the algorithm, and then we'd have needed less fiddling with test data when changing these parameters.